ament_index_cpp/README.md

```diff
@@ -0,0 +1,9 @@
+# ament_index_cpp
+
+`ament_index_cpp` is a C++ API to access the ament resource index.
+
```

Comment:
we should include the features in the README.md or Doxygen

Reply:
I just assigned myself to that. However, it is not directly related to this PR, shouldn't stop merging this